content/synchronization_model.tex

```diff
@@ -108,3 +87,25 @@
 \hline
 \end{tabular}
 \clearpage
+
+\begin{tabular}{p{0.2\textwidth} | p{0.7\textwidth}}
+Collective synchronization over \\
+\FUNC{shmem\_team\_sync}
+&
+\raisebox{-\totalheight}{\includegraphics[width=0.8\textwidth]{figures/sync}}
```

Comment:
Figures are spilling into the margin. Can we try `width=\linewidth`?

Reply:
May need to remove one of the PEs from the figure in order to make the figure still readable.